docs/application-setup.md

```diff
@@ -0,0 +1,107 @@
+## Application setup
+
+Prior to the implementation of Luigi, you need to set up your application. This document shows you how to set up a web application using the Luigi micro front-end framework.
+
+## Intitial steps
+
+1. Add Luigi npm packages to your project dependencies. 
+2. Give Luigi exclusive control over the entry `index.html` file. If you have a single page application, transfer it to a two-page application. Make sure the existing router doesn't interfere.
+3. Define your Luigi [navigation configuration](navigation-configuration.md).
+4. Start your server to run the application.
+
+## Command examples
+
+The examples on this page demonstrate commands that perform each of the necessary steps to set up your application. Each set of commands is grouped by the framework on which you execute it.
+
+### Application setup for an application not using a framework
+
+>**NOTE:** A development server capable of hosting Single Page Applications must be installed. We recommend Live Server.
```

Comment:
`You need a development server capable of hosting Single Page Applications. The recommended server is Live Server.`